src/js/providers/tracks-mixin.js

```diff
@@ -430,11 +430,11 @@ define(['../utils/underscore',
     //////////////////////
 
     function _addCueToTrack(track, vttCue) {
-        if (!utils.isEdge() || !window.TextTrackCue) {
+        if (!utils.isEdge() || !utils.isTrident() || !window.TextTrackCue) {
```

Comment:
No. The fix is to not add `VTTCue` to `window` because that makes Shaka use VTTCues when the browser does not support adding JavaScript object instances to TextTracks.

This just makes sure we don't make the same mistake in IE11.